source/overview.rst

```diff
@@ -165,3 +165,23 @@ The WEC-Sim source code is located in the ``$WECSIM`` directory, but ``wecSim``
 .. Note::
 	WEC-Sim simulations should always be executed from the MATLAB Command Window, not from Simulink.
 
+Software Tests
+--------------------
+This section describes the software tests developed to test the WEC-Sim source code. 
+WEC-Sim includes continuous integration software tests. The tests are run each time changes are made to the repository, and are designed to ensure that the code is performing as expected. New tests are developed each time new functions are added or modified.
+Refer to MATLAB's `unit test <https://www.mathworks.com/help/matlab/matlab-unit-test-framework.html?s_tid=CRUX_lftnav>`_ and `continuous integration <https://www.mathworks.com/help/matlab/matlab_prog/continuous-integration-with-matlab-on-ci-platforms.html>`_ documenation for more information. 
```

Comment:
* typo in documentation

Reply:
@akeeste I fixed this typo with https://github.com/WEC-Sim/WEC-Sim/pull/434/commits/ef940354b6a92d6843deb8002fb05290098ff9b6 and will go ahead and merge this PR